### DISCUSSION TITLE
Error code 500 on a 404 not found page on deployment :/

### DISCUSSION THREAD

**Discussion by GoldenAgeMike** [vercel/vercel#6616] [Help]
I deployed my app on Vercel few months ago, all the nav links were working fine. Past month I had made some changes in the code and now my dashboard link is not working (on deployment only, works fine locally). Moreover the error it throws is a 500 on that 404 page. I've tried redeploying with build cache cleared, but nothing changed and it still doesnt work.
Please help !!!

**Comment by leerob** [comment]
Hey! To assist this discussion further, would you mind providing more information?

- URL to your deployed application on Vercel
- Public link to a GitHub repo where we can look at the code

If you cannot share the source code, can you create a small, reproducible example of the issue? 🙏 Thank you!

**Reply by GoldenAgeMike** [reply]
Hi there leerob, I kind of solved the issue by removing getserverside and instead using getstaticprops. I don't know what was wrong with my code. I would love to learn what was i doing wrong  in my approach to this. A big fan of your youtube content ✌️.
This is the project URL: https://gam-delta.vercel.app/
The code in question : https://stackoverflow.com/questions/68871261/error-code-500-on-a-404-not-found-page-when-deploying-on-vercel
